### PR TITLE
VPNHotspot: Translate: Update zh-rTW Translate (09/04)

### PR DESCRIPTION
**Pull request by bluehomewu**
* Fix some typo and modify some word to Traditional Chinese

Signed-off-by: Edward Wu <bluehome.wu@gmail.com>

### CODE REVIEW

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -180,9 +180,9 @@
     <string name="noisy_su_failure">發生錯誤，請檢查偵錯資訊。</string>
     <string name="permission_missing">權限不足</string>
 
-    <string name="configuration_view">WIFI 組態</string>
+    <string name="configuration_view">WiFi 組態</string>
```
Comment (Mygod) [inline mobile/src/main/res/values-zh-rTW/strings.xml:183]: Use `Wi\u2011Fi` like in English?

Comment (bluehomewu) [reply mobile/src/main/res/values-zh-rTW/strings.xml:183]: Well...I think is good rn.

Comment (bluehomewu) [reply mobile/src/main/res/values-zh-rTW/strings.xml:183]: @Mygod Can you merge PR?

Comment (Mygod) [reply mobile/src/main/res/values-zh-rTW/strings.xml:183]: Please use `Wi\u2011Fi`.

**Pull request merged**